prov/efa/src/rdm/rxr_pkt_type_misc.c

```diff
@@ -305,8 +305,8 @@ void rxr_pkt_handle_readrsp_sent(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_en
 	rx_entry->window -= data_len;
 	assert(rx_entry->window >= 0);
 	if (rx_entry->bytes_sent < rx_entry->total_len) {
-		if (rx_entry->desc[0] || efa_is_cache_available(rxr_ep_domain(ep)))
```

Comment:
`rxr_prepare_desc_send` used to do more than call `rxr_op_entry_try_fill_desc`. 

There used be be `tx_iov_idx` and `tx_iov_offset` inside `rxr_op_entry`. For which `rxr_prepare_desc_send` call `rxr_iov_locate_pos` to set.

In that case, we need to call `rxr_prepare_desc_send` for `rx_entry->desc[0]` and `tx_entry->desc[0]`.

We later removed `tx_iov_idx` and `tx_iov_offset` from `rxr_op_entry`. They are now calculated on the fly in `rxr_pkt_init_data_from_op_entry`, and `rxr_prepare_desc_send` became simply calling `rxr_op_entry_try_fill_desc`. So that condition is not necessary any more.